.github/actions/ossf-compiler-flags-scanner/action.yaml

```diff
@@ -1,9 +1,21 @@
 # %CopyrightBegin%
 #
-# SPDX-FileCopyrightText: Copyright Ericsson AB 2023-2025. All Rights Reserved.
-#
 # SPDX-License-Identifier: Apache-2.0
 #
+# Copyright Ericsson AB 2024-2025. All Rights Reserved.
```

Comment:
the year changed here, is this ok and on purpose or something to review

Reply:
I've updated some years so that they are more correct. I hope that the updates are correct. This I think was a copypaste error from some other `action.yaml` file.